2015/12/08/stackstrom-1-2-release-announcement/index.html

```diff
@@ -373,57 +373,57 @@ <h4>Serving api and auth off single HTTPS port</h4>
 						<section id="text-7" class="col"><div class="scn">			<div class="textwidget">
 <p><a href="http://docs.stackstorm.com/" target="_blank" rel="noopener">Documentation</a><br>
 <a href="https://github.com/StackStorm" target="_blank" rel="noopener">GitHub</a><br>
-<a href="/#community" target="_blank" rel="noopener">Community</a><br>
-<a href="/security" target="_blank" rel="noopener">Security</a></p>
+<a href="../../../../index.html#community" target="_blank" rel="noopener">Community</a><br>
+<a href="../../../../security/index.html" target="_blank" rel="noopener">Security</a></p>
 </div>
 		</div></section>					</div>
 									<div class="colm oth ">
 						<section id="text-8" class="col"><div class="scn">			<div class="textwidget">
-<p><a href="/blog">Blog</a><br>
+<p><a href="../../../../blog/index.html">Blog</a><br>
```

Comment:
At minimum we shouldn't point to `index.html` in any case as that part should be hidden 
```suggestion
<p><a href="../../../../blog">Blog</a><br>
```

Reply:
Makes sense.

The code base is plain HTML/JS/CSS, so I believe there aren't much options for resolving URL paths and for hiding the filename while still having the links work. We could technically create a global variable that yields the root directory's path and share it across all the files using a `<script>` tag, but that will only make the codebase messier and more difficult to maintain, since every `src` and `href` value would need to be injected via JS. Of course, even then the website would break on JavaScript disabled browsers.

If this codebase is a part of some kind of framework when ported to your testing/production environment - something that can handle dynamic values across the files - then I believe the solution should become much simpler.

I'm not yet an experienced developer (just about 2 years under my belt) and this is my first ever open source contribution, so please feel free to correct me wherever I'm wrong. Appreciate your comments. A big thanks for reviewing my contribution.